app/models/site.rb

```diff
@@ -159,13 +159,14 @@ def find_by_request(request)
     # @param [Partner]
     # @return [ActiveRecord::Relation<Site>]
     def sites_that_contain_partner(partner)
-      neighbourhood_ids = partner.owned_neighbourhood_ids
-      tag_ids = partner.partner_tags.pluck(:tag_id)
-
-      Site.left_outer_joins(:sites_tag, :sites_neighbourhoods)
-          .group(:id)
-          .where('neighbourhood_id in (?) or tag_id in (?)',
-                 neighbourhood_ids, tag_ids)
+      sites = Site.all.order(:name)
```

Comment:
I think your problem may be that the original code also dealt with tags as well as neighbourhoods.

Reply:
should that matter because here I compare a partner to a list of partners instead of inferring the inclusion based on shared attributes?